src/org/centos/contra/Infra/Utils.groovy

```diff
@@ -453,12 +486,17 @@ def createSSHKeyFile(String providerType, String containerName, String sshFileId
             mkdir -p "${key_store_path}"
             cp "${env.SSH_FILE_NAME}" "${key_store_path}/${providerType}.ssh"
             """
+
+            sshUser = sshUser ?: env.SSH_USERNAME
+            sshPassphrase = sshPassphrase ?: env.SSH_PASSPHRASE
         }
-        if ( env.SSH_PASSPHRASE ) {
```

Comment:
I think that this should remain as env.SSH_PASSPHRASE

Reply:
Yes, I encountered a bug during dev where parallel threads would overrite this variable. It was (and should be) fixed external to HDSL. I will revert everything related to SSH_PASSPHRASE.